topics/proteomics/tutorials/clinical-mp-1-database-generation/tutorial.md

```diff
@@ -106,7 +106,7 @@ The first workflow for the clinical metaproteomics data analysis is the Database
 >
 > 1. **Import the workflow** into Galaxy:
 >
->    {% snippet faqs/galaxy/workflows_run_trs.md path="topics/proteomics/tutorials/clinical-mp-database-generation/workflows/main_workflow.ga" title="Pretreatments" %}
+>    {% snippet faqs/galaxy/workflows_run_trs.md path="[https://usegalaxy.eu/u/galaxyp/w/wf1databasegenerationworkflow]" title="Pretreatments" %}
```

Comment:
Hi @subinamehta! Thanks for you updates. These comments from the bot are because you need to download the workflow files from Galaxy and add them to the GTN, in the workflows folder of your tutorial folder. And then just add the file path and name here.

(let me know if you need some help with that)